Source/IGListStackedSectionController.m

```diff
@@ -213,12 +213,26 @@ - (UICollectionReusableView *)dequeueReusableSupplementaryViewOfKind:(NSString *
                                                                class:(Class)viewClass
                                                              atIndex:(NSInteger)index {
     const NSUInteger offset = [self offsetForSectionController:sectionController];
-    return (UICollectionViewCell *_Nonnull)[self.collectionContext dequeueReusableSupplementaryViewOfKind:elementKind
+    return (UICollectionReusableView *_Nonnull)[self.collectionContext dequeueReusableSupplementaryViewOfKind:elementKind
                                                                                      forSectionController:self
                                                                                                     class:viewClass
                                                                                                   atIndex:(index + offset)];
 }
 
+- (UICollectionReusableView *)dequeueReusableSupplementaryViewOfKind:(NSString *)elementKind
+                                                forSectionController:(IGListSectionController <IGListSectionType> *)sectionController
+                                                             nibName:(NSString *)nibName
+                                                              bundle:(NSBundle *)bundle
+                                                             atIndex:(NSInteger)index
+{
+    const NSUInteger offset = [self offsetForSectionController:sectionController];
+    return (UICollectionReusableView *_Nonnull)[self.collectionContext dequeueReusableSupplementaryViewOfKind:elementKind
```

Comment:
> i think UICollectionReusableView is more suitable here, am i wrong?

Hmm I'm actually not sure why we're casting it in the other methods. I just tried it locally and we can remove those. I wouldn't worry about it now. Can we revert the `UICollectionReusableView` changes? We can do cleanup in another PR.